src/main/java/jalau/at18/katas/bankocr/sergio/PathFile.java

```diff
@@ -0,0 +1,17 @@
+package jalau.at18.katas.bankocr.sergio;
+
+import java.io.File;
+
+public class PathFile {
+    private File file;
+    private String pathString = new String(
```

Comment:
Is the usage of this path verified by a test? my gues is that could fail. If this code is part of main logic, it must have at least a unit test to verify it

Usually, the best practice is that reference to files are relative paths instread of full paths. Example, instead of this, relative path might be:
`src\\main\\java\\jalau\\at18\\katas\\bankocr\\sergio\\BankAccounts\\BankAccounts.txt`

Reply:
solved with relative path